ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -89,8 +89,8 @@ export default class ListHooks extends Component {
             softSearch
             tree={tree}
             onEmptySearch={
-              <Typography variant="subtitle1">
-                No items for search term {hookSearch}
+              <Typography variant="h3" component="h3" color="primary">
+                No hooks are defined {hookSearch}
```

Comment:
This is looking better, but now when there *is* a search term, for example `proj-taskcluster`, it's going to say something like
> No hooks are defined proj-taskcluster

which doesn't make a lot of sense.  How could it be made to say 

> No hooks are defined

when there's no hooks defined and

> No hooks found matching "proj-taskcluster"

When there's a search term but no matching results?

Reply:
It looks like this still only ever says "No hooks are defined" -- I don't see the other strings anywhere in the PR.

Reply:
Hello, @djmitche  I am a bit confused will the string will look like this, 

`No hooks are defined ${hooksearch}`

Reply:
That would have the same effect as what's already written.

It might be good to make a diagram or notes of what the different states are, and what the component should look like in each of those states.  Then think of a way to put the component in each of those states in your browser, so you can verify whether your changes have achieved the desired result.